src/api/types/index.ts

```diff
@@ -65,6 +65,7 @@ export type FamilyDetailResponse = Pick<
 > & {
   children: Student[];
   current_enrolment: EnrolmentResponse | null;
+  enrolments: EnrolmentResponse[];
```

Comment:
sorta makes sense to name this `prev_enrolments` since we have `current_enrolment`, but I think it's fine either way

Reply:
we're acc gonna remove `current_enrolment` for a recent design update 🥲 so it's just gonna be `enrolments` moving forward

Reply:
ok dope